test/dni_nie_validator_test.dart

```diff
@@ -8,4 +8,9 @@ void main() {
     expect('54566045V'.isValidDNI(), true);
     expect('54566045i'.isValidDNI(), false);
   });
+  test('validate CIF', () {
+    expect('E47460258'.isValidCIF(), true);
+    expect('J95486403'.isValidCIF(), true);
+    expect('F14547923'.isValidCIF(), false);
```

Comment:
can you add a test that validates that `isValidCIF` is true?

Reply:
I think they are enough, similar to the tests for DNI.